src/assets/scripts/client/aircraft/AircraftStripView.js

```diff
@@ -36,11 +36,11 @@ export default class AircraftStripView {
      * @for AircraftStripView
      * @constructor
      * @param callsign {string}  this property is a result of a function call and not directly tied to the
-     *                           `AircraftInstanceModel`, thus it is included explicitly intead of obtainined
-     *                           from the `AircraftInstanceModel`
-     * @param aircraftInstanceModel {AircraftInstanceModel}
+     *                           `AircraftModel`, thus it is included explicitly intead of obtainined
+     *                           from the `AircraftModel`
+     * @param AircraftModel {AircraftModel}
      */
-    constructor(aircraftInstanceModel) {
+    constructor(AircraftModel) {
```

Comment:
one must be very careful with find and replace.  this time it looks like `aircraftInstanceModel` has been changed to `AircraftModel`, which is incorrect.  it should be `aircraftModel`